### ISSUE TITLE
Add error checking based on bad API being passed into class.

### ISSUE THREAD

**Issue by tlmurphy** [jmhossler/twitter-mood#3]
It would be good to add some kind of error message if a user passes in something non-compatible with our module.

**Comment by jmhossler** [comment]
yeah, I want to switch it out. It would be nice if we could just say you pass in a python twitter interface.

**Comment by jmhossler** [comment]
I'd rather us not rely on the outside type. And I would like it to change so that it doesn't rely on a specific sentiment analysis tool
